packages/keyring-eth-ledger-bridge/src/ledger-keyring.ts

```diff
@@ -540,7 +573,24 @@ export class LedgerKeyring implements Keyring {
           message,
         },
       });
-    } catch (error) {
+    } catch (error: unknown) {
+      if (error instanceof TransportStatusError) {
+        // cast error to TransportError
+        const transportError: TransportStatusError = error;
+        if (
+          transportError.statusCode === 27013 &&
+          transportError.message.includes('(denied by the user?) (0x6985)')
+        ) {
+          throw new Error('Ledger: User rejected the transaction');
+        }
+
+        if (
+          transportError.statusCode === 27264 &&
+          transportError.message.includes('Invalid data received (0x6a80)')
+        ) {
+          throw new Error('Ledger: Blind signing must be enabled');
+        }
+      }
```

Comment:
Then we can use it like:
```suggestion
      reThrowIfTransportError(error);
      
```

Yes there's a missing empty line here too (compared to your other 2 other `catch` clauses)